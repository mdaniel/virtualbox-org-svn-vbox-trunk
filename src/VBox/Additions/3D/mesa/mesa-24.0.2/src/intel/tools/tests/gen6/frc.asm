frc.sat(8)      m4<1>F          g3<4>F                          { align16 1Q };
frc(8)          g19<1>.xF       (abs)g1<0>.xF                   { align16 1Q };
frc(8)          g12<1>F         g6<8,8,1>F                      { align1 1Q };
frc(16)         g18<1>F         g9<8,8,1>F                      { align1 1H };
frc(8)          m1<1>F          g9<8,8,1>F                      { align1 1Q };
frc(16)         m1<1>F          g11<8,8,1>F                     { align1 1H };
