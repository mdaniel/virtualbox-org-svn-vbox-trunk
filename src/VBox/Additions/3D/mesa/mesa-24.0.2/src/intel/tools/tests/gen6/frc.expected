43 01 60 80 be 03 8f 20 64 00 6e 00 00 00 00 00
43 01 60 00 bd 03 61 22 20 20 00 00 00 00 00 00
43 00 60 00 bd 03 80 21 c0 00 8d 00 00 00 00 00
43 00 80 00 bd 03 40 22 20 01 8d 00 00 00 00 00
43 00 60 00 be 03 20 20 20 01 8d 00 00 00 00 00
43 00 80 00 be 03 20 20 60 01 8d 00 00 00 00 00
